styles/styles.css

```diff
@@ -14,7 +14,139 @@ body {
 }
 
 .container {
-  padding: 120px 104px 120px 104px;
+  padding: 0 104px 0 104px;
+  margin: 0 auto;
```

Comment:
I suppose it will be great, if you add max-width propperty, equal for measurments on figma file. As you can notice, all block content equal, so, i thin it should be in your container. Don't forget about width:100% also